uni/lib/controller/parsers/parser_schedule.dart

```diff
@@ -10,7 +10,6 @@ Future<List<Lecture>> parseScheduleMultipleRequests(responses) async {
   }
   return lectures;
 }
-
```

Comment:
Revert back to what it was, a \n to seperate the functions makes the code more clean